pipelines/main/launch_unsigned_jobs.yml

```diff
@@ -12,6 +12,24 @@
 # and only need to touch the webui configuration when we need to alter
 # something about the privileged steps.
 
+env:
```

Comment:
This is only needed as this repo's main branch is called "main" whereas the one of the Julia repo is called "master".

Using a default of `DIFF_FILTER_TARGET_BRANCH=""` tells the `diff-filter-buildkite-plugin` to use a PR's merge base as the reference against which changes should be compared (https://github.com/fatteneder/diff-filter-buildkite-plugin/blob/7155804f7ba73a9798b4c4225613ca6a5d7d8c12/hooks/diff-filter-impl#L13-L38).

So if the pipeline run's on forks of Julia's repo, it should do the right thing (:tm: + untested). For PRs against `julia-buildkite` we always compare against Julia's master by manually setting it in `.buildkite/hooks/post-checkout`.